package-lock.json

```diff
@@ -17,28 +17,32 @@
       }
     },
     "node_modules/@user-office-software/duo-logger": {
-      "version": "2.0.2",
-      "resolved": "https://registry.npmjs.org/@user-office-software/duo-logger/-/duo-logger-2.0.2.tgz",
-      "integrity": "sha512-SMagg/lKVAuQIgyun+ne88gpCpQQr+edEn4ZpBzEMEY+wzlkqwAI6kVG/hpju9GfbGrKTUpb34zeuU7lmlfbeg==",
+      "version": "2.1.1",
```

Comment:
We might be missing the changes in package.json

Reply:
Thanks 🙂